meeteval/viz/visualize.js

```diff
@@ -1636,28 +1681,37 @@ class CanvasPlot {
                 let tooltip = addTooltip(element, false).classed("wrap-60 alignleft", true)
 
                 tooltip.append("div").attr('align', 'left').text(value)
+                let warning_field = tooltip.append("div").attr('align', 'left').style("display", "none").text(
+                    "\nIssue while loading\n  " + settings.audio_server + value +
+                    "\nWith access to the audio file (either via audio server from the menu, or directly from the filesystem), a player will appear.\n" +
+                    "Options:\n" +
+                    " - With 'python -m meeteval.viz.file_server' you can start a process, that exposes normalized wav files on http://localhost:7777\n" +
```

Comment:
There is a `<code>` tag that formats code correctly (https://www.w3schools.com/tags/tag_code.asp)

Reply:
Done.